src/schema/objects/common_principles.yaml

```
@@ -44,6 +43,11 @@ data_type:
 
         14. `mrs` (magnetic resonance spectroscopy)
 
+        15. `phenotype` (participant-level measurement and survey data)
```

so it would be coming under a session folder if it is a dataset where there is a session folder for a subject, right?  so it would not be "participant-level" but "session-level"?  then I would just KISS:

```suggestion
        15. `phenotype` (measurement and survey data)
```

---

No, the whole idea is that this doesn't change the meaning of phenotype, it slightly broadens the notion of datatype. This does not permit phenotype directories in sessions.

---

but they are becoming (or already?) allowed at subject level so I could have `sub-1/phenotype/` in addition to e.g. `sub-1/ses-1`?  if not, then it is still located at "dataset level" (as in https://bids-specification.readthedocs.io/en/stable/modality-agnostic-files.html#phenotypic-and-assessment-data) and "participant-level" is confusing here... then

```suggestion
        15. `phenotype` (participant-level measurement and survey data at dataset level phenotype/)
```

or alike to clear it up?